pages/linux/amap.md

```diff
@@ -0,0 +1,20 @@
+# amap
+
+> Next-generation scanning tool for pentesters.
+> More information: <https://www.thc.org>.
+
+- Scan port 80 on a target host and print ASCII banner response, omit closed ports:
+
+`amap -bqv {{ip_or_hostname}} {{80}}`
```

Comment:
```suggestion
- Scan port 80 on a target host and print ASCII banner response, omit closed ports:

`amap -bq --verbose {{ip|hostname}} {{80}}`
```

Warning: [mnemonics can't be suggested as parsing `amap` man page failed: no `-b`, `-q` options found, possible broken man page](https://manned.org/amap.1).
Mandatory fix: [long option must be used if it's portable](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax).
Mandatory fix: [`or` keyword is undefined for anything expect `file_or_directory`, wrong syntax used](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths).